django_ecommerce/settings.py

```diff
@@ -20,14 +21,15 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = 'django-insecure-g%2@m997h_a#=jtwvse5yrmae@3mb#5=fak-m93!=jmf6l$()k'
+SECRET_KEY = 'django-insecure-b=+&bq(#sm9srpii+13qo!7uq!ixxtg5s1h^m^o79efd54fqk0'
```

Comment:
I think a better approach is to store this secret key in `.env` file and use it from there

Reply:
*Store secret key in `.env` file and also add `.env` file in `.gitignore`*